docs/javascript/v2/how-to-guides/set-up-video-conferencing/peerlist-optimizations.mdx

```diff
@@ -0,0 +1,78 @@
+---
+title: Handling peerlist in large room
+nav: 1.23
+---
+
+Prerequisites:
+
+**Minimum required [SDK version](../../changelog/release-notes#2023-10-04)**
+
+In case of large rooms(more than 5k peers), it does not make sense to show all the peers at once as the peers of interest would be limited. So we updated our implementation
```

Comment:
```suggestion
In case of large rooms(more than 2.5k peers), it does not make sense to show all the peers at once as the peers of interest would be limited. So we updated our implementation
```